src/tests/ftest/directory_tree.py

```diff
@@ -0,0 +1,52 @@
+"""
+  (C) Copyright 2018-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import sys
+from argparse import ArgumentParser
+
+from util.io_utilities import DirTree
```

Comment:
I'm not sure if these imports will always work in any generic virtual env. But we can cross that bridge when/if needed